todo/build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "todomvc"
 
-version in ThisBuild := "0.3.0-SNAPSHOT"
+version in ThisBuild := "0.4.0-SNAPSHOT"
```

Comment:
Yes, I left `SNAPSHOT` version to enable `fastOptJS` compilation if someone wants to try it.